### ISSUE TITLE
Scraper for TheKitchenMagpie.com

### ISSUE THREAD

**Issue by mrkitchenmagpie** [hhursev/recipe-scrapers#312]
possible to get a scraper for thekitchenmagpie.com? Sample Link: [http://thekitchenmagpie.com/air-fryer-chicken-wings/](https://www.thekitchenmagpie.com/air-fryer-chicken-wings/)

**Comment by bfcarpio** [comment]
It looks like that website has json schema available which means that you can already use our scraper if you specify the `wild_mode` parameter:

```python
>>> s = scrape_me("https://www.thekitchenmagpie.com/air-fryer-chicken-wings/", wild_mode=True)
>>> s.title()
'Air Fryer Chicken Wings - Seasoned'
>>> s.ingredients()
['12 split chicken wings ((drumettes and wingettes))', '1/3 teaspoon celery salt', '1/3 teaspoon garlic powder', '1/3 teaspoon black pepper', '1 teaspoon baking powder']
```
I just wanted to let you know in case you wanted it here and now. It is not "officially" supported because there's no dedicated code and tests to make sure it's working. Official support can certainly be added though. :)

**Comment by jayaddison** [comment]
Support for TheKitchenMagpie was added in #391, and is available in versions 13.13.1 onwards.
